src/collection.coffee

```diff
@@ -43,13 +46,20 @@ class window.Brainstem.Collection extends Backbone.Collection
   lastFetchOptions: null
   firstFetchOptions: null
 
+  model: (attrs, options) ->
+    Model = require('./model')
```

Comment:
Even though this is specific to this method, we should probably put this require on the top since this file ultimately depends on our model module.